libs/deal-ticket/src/components/deal-ticket/deal-ticket-market-amount.tsx

```diff
@@ -29,7 +29,7 @@ export const DealTicketMarketAmount = ({
     : undefined;
 
   return (
-    <div className="mb-6">
+    <div className="mb-4">
       <div className="flex items-end gap-4 mb-2">
         <div className="flex-1 text-sm">Size</div>
```

Comment:
(non-blocking): maybe, by the way, will be useful to use translation helper here?